bin/index.js

```diff
@@ -2,7 +2,6 @@
 /* global error */
 'use strict';
 
-require('shelljs/global');
```

Comment:
IIRC, the call of ```exec()``` in this file was the only need for this requirement. As I kicked those calls ```shelljs/global``` is not needed anymore IMHO. Hence, it was code cleaning. If it's needed somewhere else, let's bring it back in.

Sidenote: My personal opinion on this library isn't quite positive at all as it pollutes the global scope and it becomes harder to trace where such global functions like ```exec``` actually come from. But that's just a matter of taste. ;-)